Assets/Scripts/PointRenderer3D.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Netherlands3D.Twin
 {
-    public class BatchedMeshInstanceRenderer : MonoBehaviour
+    public class PointRenderer3D : MonoBehaviour
```

Comment:
waarom is dit gerenamed? het idee was dat je een random mesh kon gebruiken en niet alleen points